internal/check/valid_owner.go

```diff
@@ -101,14 +101,14 @@ func isEmailAddress(s string) bool {
 	return err == nil
 }
 
-func isGithubTeam(s string) bool {
+func isGitHubTeam(s string) bool {
```

Comment:
This is an internal function, changing it seems harmless

Reply:
it's ok as you changed also the references